docker/Dockerfile.build-linux

```diff
@@ -13,7 +13,8 @@ RUN apt-get update \
   && apt-key adv --keyserver keyserver.ubuntu.com --recv 15CF4D18AF4F7421 \
   && add-apt-repository "deb [arch=amd64] http://apt.llvm.org/xenial/ llvm-toolchain-xenial-8 main" \
   && apt-get update \
-  && apt-get -y install clang-8 clang-tools-8 build-essential pkg-config autoconf sudo git python docker-ce-cli
+  && apt-get -y install clang-8 clang-tools-8 build-essential pkg-config autoconf sudo git python docker-ce-cli \
+  && apt-get -y install pciutils
```

Comment:
you can just add `pciutils` at the end of previous line :) also, seems like `sudo` is unnecessary here. everything is running as root.

Reply:
👍 Condensed onto one line, will leave the `sudo` thing just to keep this patch simple